nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -163,11 +163,11 @@ function parseSubscriber(line: string) {
   };
 }
 
-function parseSubscriberFile(fileObj: File) {
+function parseSubscriberFile(props: {fileObj: File, delete: boolean}) {
```

Comment:
```suggestion
function parseSubscriberFile(fileObj: File, delete: boolean) {
```

Don't use an object as parameter to simulate named parameters - this is a pattern that we haven't used throughout the NMS, and this is more specific to React components.